src/integration/common/header.js

```diff
@@ -12,10 +12,18 @@ export default () => {
       const navigationItemEls = document.querySelector(
         'header nav ul > li > a',
       );
-
       expect(navigationItemEls).toBeInTheDocument();
     });
 
+    const navigationLinks = document.querySelectorAll('header nav a');
+    navigationLinks.forEach(navigationLink => {
+      it(`I can see a navigation link: ${navigationLink.textContent}`, () => {
+        expect(navigationLink).toBeInTheDocument();
+        expect(navigationLink.textContent).toBeTruthy();
+        expect(navigationLink.getAttribute('href')).toMatchSnapshot();
+      });
```

Comment:
Worth adding `expect(navigationLink.textContent).toMatchSnapshot();`? Or is the thinking updated translations may impact this?

Reply:
It is already in the snapshot (as it has been included in the `it()`) -> e.g. https://github.com/bbc/simorgh/pull/6282/files/3d85144c07d59299c89c895694b75c4f24c1ff86#diff-5f91e4ab11cd08e4cfe934ac2ef30167R110

Reply:
💣